src/content/changes/highlights/2022-02-11.md

```diff
@@ -9,7 +9,7 @@ owner:
 
 ## User interfaces
 
-- The `kubectl gs login` command has been improved in several ways in [kubectl-gs v2.1.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v2.1.0/). You can now also use `--self-contained` when logging in to a management cluster. The new `--keep-context` flag allows to avoid changing your current kubectl context. And token renewal should be more robust now.
+- The `kubectl gs login` command has been improved in several ways in [kubectl-gs v2.1.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v2.1.0/). You can now also use `--self-contained` when logging in to a management cluster. This flag specifies the output file path for a self-contained kubectl configuration file. If provided, the certificate data will be written to an external kubectl configuration file instead of the default one. Please be aware that this starts the login process from the beginning regardless of existing contexts. The new `--keep-context` flag allows to avoid changing your current kubectl context. And token renewal should be more robust now.
```

Comment:
```suggestion
- The `kubectl gs login` command has been improved in several ways in [kubectl-gs v2.1.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v2.1.0/). You can now use `--self-contained` when logging in to a management cluster to output the login configuration to a self-contained `kubeconfig` file. The new `--keep-context` flag allows to avoid changing your current kubectl context. And token renewal should be more robust now.
```